### PR TITLE
API Portal February 23 release tasks

### PR DESCRIPTION
**Pull request by andreamussap**
Feature branch for API Portal February 23 release.

http://pr-2556.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20230228_apip_relnotes/

* Replaces #2533 
* Refers to Jiras: RMF-358

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | a4055e2 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2556.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20230228_apip_relnotes.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ It is important, especially when upgrading from an earlier version, to be aware
 
 ### New settings available for conditionally switching to single pane layout if the volume of endpoints are larger than the default.
 
-To improve performance when viewing APIs that have a large number of swagger endpoints, we have added a configuration option in the Joomla Admin interface to allow users to define a maximum number of endpoints. When this maximum is exceeded, the layout will be automatically switched to single pane as this view performs better with large volumes of endpoints. See more in [Customizing the API catalog and details views](/docs/apim_administration/apiportal_admin/customize_apicatalog_overview)
+To improve performance when viewing APIs that have a large number of swagger endpoints, we have added a configuration option in the Joomla Admin interface to allow users to define a maximum number of endpoints. When this maximum is exceeded, the layout will be automatically switched to single pane as this view performs better with large volumes of endpoints. See more in [Customizing the API catalog and details views.](https://docs.axway.com/bundle/axway-open-docs/page/docs/apim_administration/apiportal_admin/customize_apicatalog_overview/index.html)
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20230228_apip_relnotes.md:48]: @jowens-axway Why have you changed the relative path to the absolut path in the production portal?

Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20230228_apip_relnotes.md:48]: The relative path that was there led to a 404 not found, i couldn't get it working properly, so ended up putting in the absolute path. Meant to raise this with you actually as i know using the absolute is not the way to go.... can you fix it?

Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20230228_apip_relnotes.md:48]: it should point to /docs/apim_administration/apiportal_admin/customize_apicatalog_overview

**Pull request merged**